.ci/templates/windows-devtools.yml

```diff
@@ -1,3 +1,7 @@
+parameters:
```

Comment:
I think that this part of the change is causing the issue.  When you list parameters, you need to list all of them :-(